Assets/AnkrDemo/Scripts/WebGLHeaderWalletsPanel.cs

```diff
@@ -0,0 +1,71 @@
+using System;
+using System.Collections.Generic;
+using AnkrSDK.Data;
+using AnkrSDK.Examples.UseCases.WebGlLogin;
+using Demo.Scripts;
+using Newtonsoft.Json;
+using UnityEngine;
+
+namespace AnkrDemo.Scripts
+{
+	public class WebGLHeaderWalletsPanel : MonoBehaviour
+	{
+		[SerializeField]
+		private GameObject _panel;
+		[SerializeField]
+		private GameObject _buttonPrefab;
+		[SerializeField]
+		private WalletItem[] _wallets;
+		private List<HeaderWalletButton> _buttons;
+		public Action<Wallet> ConnectTo;
+
+		private void Start()
+		{
+			_buttons = new List<HeaderWalletButton>();
```

Comment:
move it from start to declaration

Reply:
or if you want it to be initialized by unity lifecycle move it to awake and set its capacity to _wallets.Count